docs/mp/jaxrs/05_jaxrs-applications.adoc

```diff
@@ -90,3 +90,19 @@ public class MyResource {
 This approach effectively moves the scope of `Application` subclass instances to
 request scope in order to access the correct subclass for the resource method being
 executed.
+
+== Injection Managers in Helidon
+
+Jersey does not currently provider support for multiple `Application` subclasses.
+As a result, it creates a single internal _injection manager_ for your entire application,
```

Comment:
Tiny suggestion: Maybe consider changing "application" here (and in lines 101 and 108) to "server" or something like that, just to avoid possible confusion with `Application` which appears frequently in these paragraphs.

Reply:
There's a note at the top of that document that explains that.

Reply:
Ah, yes. It didn't change so didn't show up in the diff and I didn't open the whole file. Looks good.